sections/security/avoideval.chinese.md

```diff
@@ -0,0 +1,22 @@
+# 避免JS eval语法
+
+### 一段解释
+
+`eval()`，`setTimeout()`和`setInterval()`是全局方法，经常在Node.js中被使用，它接收一个字符串参数，代表一个JavaScript表达式，语句，或者语句序列。使用这些函数的安全问题是, 不受信任的用户输入可能会发现进入代码执行的方式，从而导致危害服务器，因为评估用户代码实质上允许攻击者执行任何他可以的操作。对于这些用户输入可以传递给函数并执行的方法，建议重构代码，而不依赖于它们的使用。
+
+### 代码示例
+
+```javascript
+// 攻击者可能输入的恶意代码示例
+userInput = "require('child_process').spawn('rm', ['-rf', '/'])";
```

Comment:
I don't know how this is done in the original version, but adding the variable declaration is encouraged, I think.
```suggestion
const userInput = "require('child_process').spawn('rm', ['-rf', '/'])";
```